src/sentry/tasks/relay.py

```diff
@@ -284,7 +287,12 @@ def compute_configs(organization_id=None, project_id=None, public_key=None):
             configs[key.public_key] = None
     elif project_id:
         for key in ProjectKey.objects.filter(project_id=project_id):
-            configs[key.public_key] = compute_projectkey_config(key)
+            if projectconfig_cache.exists(key):
```

Comment:
This might be premature optimization, but since we're potentially dealing with a lot of keys, could we do a batch redis scan here?